e2e/_suites/fleet/fleet.go

```diff
@@ -361,7 +378,7 @@ func (fts *FleetTestSuite) anAgentIsDeployedToFleetWithInstallerAndFleetServer(i
 // we are using the Docker client instead of docker-compose because it does not support
 // returning the output of a command: it simply returns error level
 func (fts *FleetTestSuite) getContainerName(i installer.ElasticAgentInstaller, index int) string {
-	return fmt.Sprintf("%s_%s_%s_%d", i.Profile, i.Image, common.ElasticAgentServiceName, index)
```

Comment:
Same thing here, in the other branch we use an `Inspect` function to populate the service metadata so that we can just reference that rather than trying to piece together the container name as these would differ between docker and k8s